mmv1/templates/terraform/custom_expand/gke_hub_membership.erb

```diff
@@ -0,0 +1,22 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2021 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d TerraformResourceData, config *Config) (interface{}, error) {
+	if strings.HasPrefix(v.(string), "//container.googleapis.com/") {
```

Comment:
Can we change this guard to just `//`? `//container.googleapis.com/` is the only environment most users will interact with, but isn't necessarily the only environment.

Reply:
should we revert?

Reply:
Seems fine to fix forward